src/www/ui/page/AboutPage.php

```diff
@@ -50,15 +50,49 @@ function __construct()
    * @return Response
    */
   protected function handle(Request $request)
-  {
+  {   
     $vars = array(
         'licenseCount' => $this->licenseDao->getLicenseCount(),
         'project' => _("FOSSology"),
-        'copyright' => _("Copyright (C) 2007-2014 Hewlett-Packard Development Company, L.P.<br>\nCopyright (C) 2014-2015 Siemens AG."),
+        'copyright' => _("Copyright (C) 2007-2014 Hewlett-Packard Development Company, L.P.<br>\nCopyright (C) 2014-2016 Siemens AG."),
     );
+    
+    if (\Fossology\Lib\Auth\Auth::isAdmin()) {
+      $latestRelease = $this->curlGit('releases/latest');
+      $since = '?since=' . date('Y-m-dH:i:sZ', time() - 3600 * 24 * 30);
+      $commits = $this->curlGit('commits' . $since);
+      $commit = substr($commits[0]['sha'],0,6);
```

Comment:
When visiting `http://localhost:8081/repo/?mod=about` with enabled debug information I get the warning
```
Notice: Undefined offset: 0 in /usr/local/share/fossology/www/ui/page/AboutPage.php on line 64
```

--

![tmp3](https://cloud.githubusercontent.com/assets/1187050/14456931/c80bbc62-00a7-11e6-909a-0a751dfb56d3.png)